docs/css-customization.md

```diff
@@ -373,3 +281,236 @@ This is the button to open the Webchat, if you want to modify its color you have
 }
 
 ```
+## Webchat templates messages customization
+
+Our Webchat ships with message templates out of the box (Quick replies, Buttons, Image Galleries, Media, Lists) these elements can be also customized to meet design guidelines even more!
+
+Here are the classes that you can use to modify it:
+
+### Quick Replies
+
+* *webchat-quick-reply-template-root*  
+This class contains the whole Quick Reply element, you can modify its size for example.
+```CSS
+[data-cognigy-webchat-root] .webchat-quick-reply-template-root {
+width: 500px;
+height: 500px;
+}
+
+```
+* *webchat-quick-reply-template-header-message*  
+This is the header message of the Quick Reply.
+```CSS
+[data-cognigy-webchat-root] .webchat-quick-reply-template-header-message { 
+  border:2px solid  #fffffe;
+  border-radius: 0;
+  text-align: center;
+  background: #e3f6f5;
+  color: #272343;
+  animation: "Some animation" ;
+}
+
+```
+* *webchat-quick-reply-template-replies-container*  
+This is the element holding all Quick Replies, you can change the way they are shown. Like a column.
```

Comment:
"Like a column." is not a proper sentence. You could write "This is the element holding all Quick Replies, you can change the way they are shown, e.g. as a column.

Reply:
fixed